src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -211,17 +217,29 @@ protected override void FileList_SelectionChanged(object sender, SelectionChange
 				presenter!.Background = this.Resources["ListViewItemBackgroundSelected"] as SolidColorBrush;
 			}
 
-			if (SelectedItems?.Count == 1 && SelectedItem?.PrimaryItemAttribute is StorageItemTypes.Folder && openedFolderPresenter != FileList.ContainerFromItem(SelectedItem))
+			if (SelectedItems?.Count == 1 && SelectedItem?.PrimaryItemAttribute is StorageItemTypes.Folder)
 			{
-				if (UserSettingsService.FoldersSettingsService.ColumnLayoutOpenFoldersWithOneClick)
+				// Close any open folder
```

Comment:
```suggestion
				// Prevents selected folders from opening if the user is currently dragging the selection rectangle (#13418)
```

Reply:
Can I clarify that it only affects the first selected item?

Reply:
Either way is fine with me